comid/integregisters_test.go

```diff
@@ -0,0 +1,230 @@
+// Copyright 2024 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package comid
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/veraison/swid"
+)
+
+func prepareRegister(iType string) (*IntegReg, error) {
+	tv := "12435465869790801243546586979080124354658697908"
```

Comment:
It's quite weird to have the sha384 output as a string.  suggest using MustHexDecode("...").  But more importantly, why do we need to vary the entries?  Are they checked somewhere?  From a quick scan, it doesn't seem to be the case.